### PR TITLE
Support downloading secrets in YAML format

### PR DESCRIPTION
**Pull request by Piccirello**

### CODE REVIEW

File: pkg/cmd/secrets.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ func downloadSecrets(cmd *cobra.Command, args []string) {
 		format = "json"
 	}
 
-	validFormats := []string{"json", "env"}
+	validFormats := []string{"json", "env", "yaml"}
```
Comment (rgmvisser) [inline pkg/cmd/secrets.go]: NIT: use typealias for this?

**Review by rgmvisser — DISMISSED** [review]

File: pkg/cmd/secrets.go (hunk 1 above)
Comment (Piccirello) [reply pkg/cmd/secrets.go]: Can you add an example of what you mean? I'm not entirely sure

Comment (rgmvisser) [reply pkg/cmd/secrets.go]: Not sure if we already talked about this or not, but this is a pretty good read: https://alehatsman.com/posts/golang_type_declaration_abuse.html

Comment (Piccirello) [reply pkg/cmd/secrets.go]: Added a type `SecretsFormat`

File: pkg/cmd/secrets.go [hunk 2]
```diff
@@ -279,8 +284,10 @@ func downloadSecrets(cmd *cobra.Command, args []string) {
...
 			utils.HandleError(err, "Unable to parse download file path")
 		}
-	} else if format == "env" {
+	} else if format == http.ENV {
 		filePath = filepath.Join(".", "doppler.env")
+	} else if format == http.YAML {
+		filePath = filepath.Join(".", "doppler.yaml")
 	} else {
 		filePath = filepath.Join(".", "doppler.json")
 	}
```
Comment (rgmvisser) [inline pkg/cmd/secrets.go]:
```
func (s SecretsFormat) OutputFile() string {
	return [...]string{"doppler.json", "doppler.env", "doppler.yaml"}[s]
}
```
?

File: pkg/http/api.go [hunk 3]
```diff
@@ -135,17 +158,20 @@ func RevokeAuthToken(host string, verifyTLS bool, token string) (map[string]inte
...
 	headers := apiKeyHeader(apiKey)
-	if jsonFlag {
+	if format == JSON {
 		headers["Accept"] = "application/json"
+	} else if format == YAML {
+		headers["Accept"] = "text/yaml"
 	} else {
 		headers["Accept"] = "text/plain"
 	}
+
```
Comment (rgmvisser) [inline pkg/http/api.go]:
```
func (s SecretsFormat) MimeType() string {
	return [...]string{"application/json", "text/plain", "text/yaml"} [s]
}
```
?

**Review by rgmvisser — DISMISSED** [review]
Nice new tests!

**Review by rgmvisser — APPROVED** [review]

**Pull request merged**